packages/css-data/src/__generated__/property-value-descriptions.ts

```diff
@@ -494,10 +494,54 @@ export const propertiesGenerated = {
   viewTransitionName: "Sets a custom name for a transition effect",
   whiteSpaceCollapse:
     "Specifies how white space inside an element is collapsed",
+  "-webkit-box-orient":
+    '"Controls the direction of flex items in a flex container."',
```

Comment:
The prompt was trying to get rid of the additional quotes inside of single quotes, but seems like its not working any more. We need to either tweak the prompt again or get rid of the quotes manuallly